### PR TITLE
A modular approach

### PR DESCRIPTION
**Pull request by noticeeverything**
This doesn't necessarily need to be merged. But I wanted to provide an example of a more modular approach. I kinda just made this up, so it's by no means the definitive "way to do things".

### CODE REVIEW

**Review by santese — APPROVED** [review]
@noticeeverything looks good! This approach is a lot cleaner and more intuitive

File: __fixtures__/app-metadata.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CalebAlbers) [inline __fixtures__/app-metadata.xml:1]: Looks like this is a carry-over from the Okta lib?

File: src/lib/events/events.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     *  example minimum keys for updateArray
+     * [
+     *   { id: 123456760, billed: true},
+     *   { id: 123456761, billed: true, label_ids: []},
+     *   { id: 123456762, billed: true, label_ids: [], project_id: 1234567 },
+     *   { id: 123456763, billed: true, label_ids: [], project_id: 1234567 },
+     *   { id: 123456764, billed: true, label_ids: [], project_id: 1234567 },
+     * ]
+     */
+    async bulkUpdate(updateArray: any[]): Promise<TimelyBulkUpdateEventsReturn[]> {
```
Comment (CalebAlbers) [inline src/lib/events/events.ts:46]: Not a blocker, but @santese we should add a type here

**Review by CalebAlbers — APPROVED** [review]

File: src/lib/events/events.ts (hunk 2 above)
Comment (santese) [reply src/lib/events/events.ts:46]: @CalebAlbers , I can rename the interface to more of a shared name, the Bulk Event Actions like Delete and Update,  have the same return

Comment (santese) [reply src/lib/events/events.ts:46]: Actually I'm blind , just realized what you pointed out , yeah let me add a type for that

**Pull request merged**